### ISSUE TITLE
Database Migration fails

### ISSUE THREAD

**Issue by OliverGrimsley** [Laragear/WebAuthn#3]
### PHP & Platform

PHP 8.1.7 (in Valet) (MAC OS dev)

### Laravel version

9.17.0

### Authenticator type

not applicable

### OS and Browser versions

MAC OS Monterrey, FireFox (current)

### Have you done this?

- [X] I am willing to share my stack trace and logs
- [X] I can reproduce this bug in isolation (vanilla Laravel install)
- [ ] I can suggest a workaround as a Pull Request

### Expectation

For the database migration to complete without error.  I just ginned up a totally new clean install, installed breeze, created one user, then tried, same issue.  MySQL:  10.7.3-MariaDB - Homebrew

### Description

I get this error:

 Illuminate\Database\QueryException 

  SQLSTATE[42000]: Syntax error or access violation: 1059 Identifier name 'webauthn_credentials_authenticatable_type_authenticatable_id_index' is too long (SQL: alter table `webauthn_credentials` add index `webauthn_credentials_authenticatable_type_authenticatable_id_index`(`authenticatable_type`, `authenticatable_id`))

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:742
    738▕         // If an exception occurs when attempting to run a query, we'll format the error
    739▕         // message to include the bindings with SQL, which will make this exception a
    740▕         // lot more helpful to the developer instead of just the database's errors.
    741▕         catch (Exception $e) {
  ➜ 742▕             throw new QueryException(
    743▕                 $query, $this->prepareBindings($bindings), $e
    744▕             );
    745▕         }
    746▕     }

      +9 vendor frames 
  10  database/migrations/2022_06_17_161949_create_webauthn_credentials.php:25
      Illuminate\Support\Facades\Facade::__callStatic("create")

      +22 vendor frames 
  33  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

### Reproduction

```php
not applicable
```

### Stack trace & logs

```shell
Illuminate\Database\QueryException 

  SQLSTATE[42000]: Syntax error or access violation: 1059 Identifier name 'webauthn_credentials_authenticatable_type_authenticatable_id_index' is too long (SQL: alter table `webauthn_credentials` add index `webauthn_credentials_authenticatable_type_authenticatable_id_index`(`authenticatable_type`, `authenticatable_id`))

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:742
    738▕         // If an exception occurs when attempting to run a query, we'll format the error
    739▕         // message to include the bindings with SQL, which will make this exception a
    740▕         // lot more helpful to the developer instead of just the database's errors.
    741▕         catch (Exception $e) {
  ➜ 742▕             throw new QueryException(
    743▕                 $query, $this->prepareBindings($bindings), $e
    744▕             );
    745▕         }
    746▕     }

      +9 vendor frames 
  10  database/migrations/2022_06_17_161949_create_webauthn_credentials.php:25
      Illuminate\Support\Facades\Facade::__callStatic("create")

      +22 vendor frames 
  33  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
```

### Are you a Patreon supporter?

No, don't give priority to this

**Comment by DarkGhostHunter** [comment]
My old friend index is too large. Gonna fix it in no time, wait there. 

Italo Baeza C.

**Comment by DarkGhostHunter** [comment]
Fixed on 1.1.1. Delete the migration and publish it again (or just change the index name).
